controllers/common/common_status_update.go

```diff
@@ -236,6 +235,16 @@ func GetClusterDecisions(
 		}
 	}
 
+	log.Info("Sorting placements", "RootPolicyName", rootPolicy.Name, "Namespace", rootPolicy.Namespace)
```

Comment:
This shouldn't be an info log message.
```suggestion
	log.V(2).Info("Sorting placements", "RootPolicyName", rootPolicy.Name, "Namespace", rootPolicy.Namespace)
```